### PR TITLE
Add install target with configurable prefix

### PR DESCRIPTION
**Pull request by aniongithub**
Add install target

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -23,3 +27,9 @@ clean:
 	rm -f test
 	rm -f *.o
 
+install: $(LIB)
+	install -d $(DESTDIR)/$(PREFIX)/lib/
```
Comment (cqbqdd11519) [inline Makefile]: Thank you for the contribution!
But I think it should be `$(DESTDIR)$(PREFIX)/lib/` without a slash between `DESTDIR` and `PREFIX`. isn't it?

Comment (aniongithub) [reply Makefile]: Not sure, does `$(DESTDIR)` have a trailing `/`? I think an extra `/` won't make a difference, but either way seems to work fine for me!

Comment (cqbqdd11519) [reply Makefile]: I mean, to match line 32-34, without the slash seems better

Comment (aniongithub) [reply Makefile]: Updated the PR!

**Review by cqbqdd11519 — APPROVED** [review]

**Pull request merged**